spec/vm_spec.cr

```diff
@@ -134,5 +136,45 @@ describe "VM -" do
       vm.print_ast output
       output.to_s.should eq "Expressions\nCall\nConst\nStringLiteral|Hello world!\n"
     end
+
+    describe "#run(*programs)" do
+      it "takes files, filepaths or IOs; and (attempts to) run them" do
```

Comment:
just for completeness, could you add a spec around what happens when one of the files/filepaths/IOs _can't_ be run? What should the expected behavior be?